src/operation/bundle.rs

```diff
@@ -69,36 +71,25 @@ impl From<mpc::Message> for BundleId {
 #[derive(Wrapper, WrapperMut, Clone, PartialEq, Eq, Hash, Debug, From)]
 #[wrapper(Deref)]
 #[wrapper_mut(DerefMut)]
-#[derive(StrictType, StrictEncode, StrictDecode)]
-#[strict_type(lib = LIB_NAME_RGB_COMMIT)]
+#[derive(StrictType, StrictDumb, StrictEncode, StrictDecode)]
+#[strict_type(lib = LIB_NAME_RGB_COMMIT, dumb = Self(NonEmptyOrdSet::with(OpId::strict_dumb())))]
 #[cfg_attr(
     feature = "serde",
     derive(Serialize, Deserialize),
     serde(crate = "serde_crate", transparent)
 )]
-pub struct InputMap(Confined<BTreeMap<Vin, OpId>, 1, U16MAX>);
+pub struct InputOpids(NonEmptyOrdSet<OpId, U16MAX>);
```

Comment:
This change introduce a double-spending issue into the consensus, and doesn't make any sense.

- if you require all transitions to be present, there is no need for the TransitionBundle at all: it adds nothing. Also, you kill the multi-party transaction use case privacy: payjoins, coinjoins, Ark etc - and will need a new structure next to PSBT and a more complex interactive process to get RGB working there

- as of now, you do not require in cosensus for all transactions presence, thus, you have a double-spent attack in v0.11.1